py/janitor/site/templates/layout.html

```diff
@@ -23,18 +26,18 @@
         <script src="/_static/chart.js"></script>
         <script src="/_static/janitor.js"></script>
         {% include "analytics.html" ignore missing without context %}
-        <div class="{% if layout != 'plain' %}document{% else %}plain-document{% endif %}">
+        <div class="{% if self.sidebar() | trim | length > 0 %}document{% else %}plain-document{% endif %}">
+            {% if self.sidebar() | trim | length > 0 %}
```

Comment:
It should rather error if you've selected a non-plain layout but have not provided a sidebar. This was the reasoning with this originally being two different templates.

The new logic makes it too easy to forget the sidebar and then end up with a plain layout by accident.

Reply:
Gotcha - in that case, will close the MR